snapcraft/storeapi/_sca_client.py

```diff
@@ -0,0 +1,346 @@
+import json
+import os
+import urllib.parse
+
+import requests
+from simplejson.scanner import JSONDecodeError
+
+from ._client import Client
+from ._status_tracker import StatusTracker
+
+from . import errors
+from . import logger, _macaroon_auth
+from . import constants
+
+
+class SCAClient(Client):
+    """The software center agent deals with managing snaps."""
+
+    def __init__(self, conf):
+        super().__init__(conf, os.environ.get(
+            'UBUNTU_STORE_API_ROOT_URL',
+            constants.UBUNTU_STORE_API_ROOT_URL))
+
+    def get_macaroon(self, acls, packages=None, channels=None):
+        data = {
+            'permissions': acls,
+        }
+        if packages is not None:
+            data.update({
+                'packages': packages,
+            })
+        if channels is not None:
+            data.update({
+                'channels': channels,
+            })
+        headers = {
+            'Accept': 'application/json',
+        }
+        response = self.post(
+            'acl/', json=data, headers=headers)
```

Comment:
Compared to the old code, it seems the `acl/verify` endpoint of the API isn't used anymore? Is this done differently now or was it redundant? I'm not that familiar with this code, so I might be missing something here. Can you please clarify?

Reply:
I think perhaps that was changed some time ago?